media/js/stats.js

```diff
@@ -0,0 +1,206 @@
+$( document ).ready(function() {
+
+  $.get(dataUrl, function(d){
+      loadTagGraph(d);
+      console.log(d)
+      var active_users = [],
+      non_active_users = [];
+      for (var j = 0; j < d.new_users.length; j++) {
```

Comment:
you could replace this with d.new_users.forEach( function (element) { } ) - maybe it's nicer looking...